client/lib/common/src/data/application/application-actions.js

```diff
@@ -37,6 +37,18 @@ class ApplicationActions extends Actions {
                     throw err;
                 });
     }
+
+    fetchApplicationVersions(id) {
+        return request
+                .get(`${Services.kio.url}${Services.kio.root}/${id}/versions`)
```

Comment:
We should aim to do URL construction in `common/data/services`, so it’s easily* changeable. Will have a look later.

* Except for templates.